app/workers/reject_awaiting_references_course_choices_worker.rb

```diff
@@ -0,0 +1,9 @@
+class RejectAwaitingReferencesCourseChoicesWorker
+  include Sidekiq::Worker
+
+  def self.perform
+    CandidateInterface::GetPreviousCyclesAwaitingReferencesCourseChoices.call&.each do |application_choice|
```

Comment:
Why do we need the `&`?

```suggestion
    CandidateInterface::GetPreviousCyclesAwaitingReferencesCourseChoices.call.each do |application_choice|
```